nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -117,6 +134,12 @@ in {
       default = null;
     };
 
+    containerdConfigFile = mkOption {
```

Comment:
Not sure I understand what `containerdConfigFile` is used for, also `default = defaultContainerdConfigFile;` seems redundant with the `containerd` `configFile = mkDefault defaultContainerdConfigFile;`?

Reply:
Yeah, removed that, sorry. That was a leftover from the past. :) (ff10d964fef1a459c2bdc3493c4dbc84151adfea)